src/main/resources/swagger.yaml

```diff
@@ -0,0 +1,949 @@
+      description: A JSON Web Key (JWK) is a JSON data structure that represents a set of public keys as a JSON object.
+      operationId: requestJwk      
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                title: WebKeysConfiguration
+                required:
+                - keys
+                type: object
+                properties:
+                  keys:
+                    type: object
+                    additionalProperties:
+                      type: object                                    
+        500:
+          description: Introspection Internal Server Failed.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'   
+  /register:
+    post:
+      tags:
+      - developers
+      summary: The Client Registration Endpoint is an OAuth 2.0 Protected Resource through which a new Client registration can be requested.
+      description: Registers new client dynamically.
+      operationId: requestRegister
+      requestBody:
+        content:
+          application/json:
+            schema:
+              title: RegisterParams
+              required:
+              - redirect_uris
+              type: object
+              properties:
+                redirect_uris:
+                  type: array
+                  description: Request parameters as JSON object with data described by Connect Client Registration Specification.
+                  example:
+                  - https://client.example.org/cb
+                  items:
+                    type: string
+      parameters:
+        - name: Authorization
+          in: header
+          required: true
+          description: Access Token that is used at the Client Configuration Endpoint   
+          schema:
+            type : string      
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                title: ClientResponse
+                $ref: '#/components/schemas/ClientResponse'                                        
+        400:
+          description: Invalid parameters are provided to endpoint.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'        
+        302:
+          description: Access Denies.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+  /register(put):
```

Comment:
There no such endpoint. Please use only actual endpoints.

Reply:
Rectified, have now kept only one endpoint /register and  listed the available operations